### DISCUSSION TITLE
Should the university mail be kept as primary mail in Github for access any benifits of github developer pack ?

### DISCUSSION THREAD

**Discussion by Sakib99-cyber** [community/community#141645] [GitHub Education] [answered]
### Hi everyone,

As a student, I have benefited from the GitHub Developer Pack. Thanks for that. However, there was one thing to know. Can anyone help me with information.
As primary mail, I had university student mail. In that case, **I set Gmail Personal as the primary mail.** And I kept **university mail as secondary mail.**
In this case, as my student benefit, those offers, will there be any problem to get them ! Or if these offers have any relation with primary mail, I want to know.
Basically, the question is, **should the university mail be kept as primary?** They check the university mail to access the offers! These should clear up a little confusion.
If anyone knows, please provide a solution.

**Comment by davevad93** [comment, marked as answer]
Hi @Sakib99-cyber ,

In general, all GitHub notifications, including those related to GitHub Education, are sent to the primary email, even if it's not the academic one, although I personally advise you to set your academic email as primary and leave your other email as secondary.

**Reply by Sakib99-cyber** [reply]
Thank You.

**Reply by davevad93** [reply]
You're welcome @Sakib99-cyber , have a nice day! 👍

**Reply by Sakib99-cyber** [reply]
You do not need to keep your university email as the primary email to get the GitHub Student Developer Pack. You can use any email address as your primary email in your GitHub account settings. However, you will need to verify your student status using your university email when applying for the Student Developer Pack. 
You can change your primary email address on GitHub without affecting your eligibility for the GitHub Student Developer Pack. Once you have been approved for the pack, you will continue to receive its benefits regardless of changes to your primary email address.                                                

Copilot AI told me these words. That means there will be no problem even if you change the primary mail and give another one. what do you think @davevad93

**Reply by davevad93** [reply]
Hey @Sakib99-cyber , yes you can safely set your personal email as primary if you are approved, however when your benefits will expire (they last 2 years from the approval date) if you'll still be a student and want to reapply, set the academic email as primary and then when you are approved you can change back. I personally still keep my academic email as primary because in my institution they provide a lifetime email address.
